recipe-server/client/control_new/tests/state/session/test_selectors.js

```diff
@@ -3,17 +3,16 @@ import { List, Map, is } from 'immutable';
 import { getSessionHistory } from 'control_new/state/app/session/selectors';
 import {
   INITIAL_STATE,
+  SessionFactory,
 } from 'control_new/tests/state/session';
 
-const historyItem = (id, category = 'recipe') =>
-  new Map({ url: id, caption: 'caption', category });
-
-let lastId = 0;
-const generateFakeItems = (count, category) => {
+const itemFactory = (count, category) => {
```

Comment:
This name becomes a bit confusing since it's not actually a "factory" per se.

Maybe something like `bulkGenerateSessions`.